apis/python/src/tiledbsoma/io/ingest.py

```diff
@@ -1722,6 +1788,7 @@ def add_matrix_to_collection(
     ingest_mode: IngestMode = "write",
     use_relative_uri: bool | None = None,
     context: SOMATileDBContext | None = None,
+    schema_validation: bool = False,
```

Comment:
Missing docstring for this (and all other) parameters.  Seems like a good time to fix?  (CC: @aaronwolen)